libraries/AC_WPNav/AC_WPNav.cpp

```diff
@@ -132,6 +132,11 @@ void AC_WPNav::wp_and_spline_init(float speed_cms)
         _wp_accel_cmss.set_and_save(WPNAV_ACCELERATION);
     }
 
+    // check _wp_radius_cm is reasonable
+    if (_wp_radius_cm < WPNAV_WP_RADIUS_MIN) {
```

Comment:
I think the following process is a good one.
SET_AND_SAVE should be set even if there is no change, and sent by telemetry.
IFCHANGED, do not set unless the value is changed.

_wp_radius_cm.set_and_save_ifchanged(MAX(_wp_radius_cm, WPNAV_WP_RADIUS_MIN));

Reply:
Ahh I ddin't know about that function. It saves a few bytes so I went with it. Though functionally I think both versions are equivalent and my original version saves a bit of work each loop.